connectors/crowdstrike/connector/server_access.py

```diff
@@ -42,13 +42,17 @@ def get_hosts(self, last_model_state_id=None):
         returns:
             hosts(list): host list
         """
+        context().logger.info("Beginning import of hosts from the CrowdStrike Falcon environment.")
         hosts = []
         try:
             discover = Discover(client_id=self.client_id, client_secret=self.client_secret, base_url=self.base_url)
+            offset = 0
             result_filter = ""
+            sort = "last_seen_timestamp|asc"
             if last_model_state_id:
                 result_filter = f"last_seen_timestamp:>'{epoch_to_datetime_conv(last_model_state_id)}'"
-            host_lookup = discover.query_hosts(filter=result_filter, limit=DISCOVER_PAGE_SIZE)
+            host_lookup = discover.query_hosts(filter=result_filter, limit=DISCOVER_PAGE_SIZE, sort=sort)
```

Comment:
Lets have a log here to keep track of some meta data about the queries we are making.

We should include a "Starting Host Query" before and a "Successful Host Query" line after this call.

Reply:
For the after, we can include the "total count" as well. Just make sure that "total count" actually exist before posting it that way.

Reply:
The filter shouldn't be changed at this point either. We could post that in the "starting query"